src/config/contracts/index.ts

```diff
@@ -55,6 +69,27 @@ export const poolFacetContracts = (npid: NetworkProvider, version?: string) => {
     });
 };
 
+export const factoryFacetContracts = (npid: NetworkProvider, version?: string) => {
+    const addresses = factoryFacetAdresses(npid, version);
+    const { web3 } = getProvider(npid);
+
+    return Object.entries(addresses).map(([name, address]: [ArtifactsKey, string]) => {
+        return new web3.eth.Contract((Artifacts[name] as any).abi, address);
+    });
+};
+
+export const diamondCut = (npid: NetworkProvider) => {
```

Comment:
This name is way too generic.. Maybe call it defaultAssetPoolDiamondCut or something? Maybe even better would be to re-use the function that determines the cuts for the updates. If you don't pass it an address to update it could return the same as this, all FacetCutAction.Add actions..